src/main/java/control/ToolViewController.java

```diff
@@ -123,8 +124,9 @@ private void generateCameraShots() {
             double cameraEnd = shot.getEndCount() + shot.getEndShotPadding();
 
             shot.getTimelineIndices().forEach(index -> {
-                    CameraShot subShot = new CameraShot(shot.getName(), shot.getDescription(),
-                                                    cameraStart, cameraEnd, shot);
+                    CameraShot subShot = new CameraShot(new GeneralShotData(
```

Comment:
for readability it might be better to define this GeneralShotData on a seperate line before it is passed to the camerashot constructor